arch/arm64/boot/dts/aspeed/aspeed-bmc-amd-congo.dts

```diff
@@ -0,0 +1,289 @@
+// SPDX-License-Identifier: GPL-2.0
+// Copyright (c) 2024 AMD Inc.
+// Author: Mahesh Kurapati <mahesh.kurapati@amd.com>
+
+/dts-v1/;
+
+#include "aspeed-g7.dtsi"
+#include "dt-bindings/gpio/aspeed-gpio.h"
+#include <dt-bindings/i2c/i2c.h>
+
+/ {
+	model = "AMD Congo BMC";
```

Comment:
can we rename as PRB/VRB instead of BMC?

Reply:
changed.